libs/features/pages/conversational-learning/src/lib/conv-hero-section/conv-hero-section.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
+@Component({
+  selector: 'elewa-website-conv-hero-section',
+  standalone: true,
+  imports: [CommonModule, AppHeaderModule, TextsModule, ButtonsModule],
```

Comment:
Move this to the module, avoid having imports in the component.

Reply:
Hello, thanks for the feedback. I have moved the imports to the module.